rs/sys/src/lib.rs

```diff
@@ -16,6 +16,12 @@ pub const PAGE_SIZE: usize = 16384;
 #[cfg(not(all(target_arch = "aarch64", target_vendor = "apple")))]
 pub const PAGE_SIZE: usize = 4096;
 
+/// The size of a huge page on x86_64 on Linux.
+/// Used for a huge page allocation as a memory optimization as 
+/// it reduces the number of page faults and improves performance.
+#[cfg(not(all(target_arch = "aarch64", target_vendor = "apple")))]
```

Comment:
Is it a problem that this cfg is completely different from where you are using the variable (`#[cfg(target_os = "linux")]`)?

Reply:
Actually, will this even compile on macs, as you don't define this variable at all on macs but you have a use statement anyway?

`PAGE_SIZE` above explicitly defines it both for macs and non-macs.